src/modules/admin/components/forms/FormDefinitionDetailPage.tsx

```diff
@@ -109,52 +123,60 @@ const FormDefinitionDetailPage = () => {
                   />
                 </CommonLabeledTextBlock>
                 <CommonLabeledTextBlock title='Status'>
-                  <FormStatusText identifer={formIdentifier} />
+                  <FormStatusText identifier={formIdentifier} />
                 </CommonLabeledTextBlock>
               </Stack>
             </CommonCard>
           </Grid>
           <Grid item xs={12} md={4}>
             <CommonCard title='Actions' titleComponent='h5'>
               <Stack gap={1.5}>
-                <RootPermissionsFilter permissions='canManageForms'>
-                  <ButtonLink
-                    to={generatePath(AdminDashboardRoutes.PREVIEW_FORM_DRAFT, {
-                      identifier: formIdentifier.identifier,
-                      formId: formIdentifier.draftVersion?.id || '',
-                    })}
-                    variant='contained'
-                    fullWidth
-                    disabled={!hasDraft}
-                  >
-                    Preview / Publish Draft
-                  </ButtonLink>
-                  <EditFormButton
-                    formIdentifier={formIdentifier}
-                    text={hasDraft ? 'Edit Draft' : 'New Draft'}
-                    editorType={FormEditorType.FormBuilder}
-                    variant='outlined'
-                  />
-                  <RootPermissionsFilter permissions='canAdministrateConfig'>
-                    <EditFormButton
-                      formIdentifier={formIdentifier}
-                      text={hasDraft ? 'Edit Draft (JSON)' : 'New Draft (JSON)'}
-                      editorType={FormEditorType.JsonEditor}
-                      variant='outlined'
-                    />
-                  </RootPermissionsFilter>
-                </RootPermissionsFilter>
                 <ButtonLink
                   to={generatePath(AdminDashboardRoutes.PREVIEW_FORM, {
                     identifier: formIdentifier.identifier,
                     formId: formIdentifier.displayVersion.id,
                   })}
-                  variant='outlined'
+                  variant={hasDraft ? 'outlined' : 'contained'}
                   fullWidth
                   disabled={!isPublished}
                 >
-                  Preview Published
+                  View Published Form
                 </ButtonLink>
+                {isPublished && (
+                  <Typography variant='caption'>
+                    Published on {parseAndFormatDate(publishedOn)}{' '}
+                    {publishedBy && `by ${publishedBy.name}`}
+                  </Typography>
+                )}
```

Comment:
We have the _exact_ dates and times down in the version history table, so it might be more useful to show a relative date here `formatRelativeDateTime(parseHmisDateString(publishedOn))`. Since it seems like it's more for understanding, at a glance, how old the form is.

Reply:
Sounds good.

I noticed while clicking around dev that this displays a publication date for forms like Enrollment and Client. It looks like it displays the date of the last time the form was updated, for example using seed_definitions. Is this okay?

Reply:
Eh, not ideal, but it's OK for now